Python/Product/ExternalProfilerDriver/DWJsonModel.cs

```diff
@@ -0,0 +1,165 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.PythonTools.Profiling.ExternalProfilerDriver
+{
+    // Most numerical integer data is represented in dwjson as a 64-bit unsigned integer
+    // Due to JSON limitations, this is represented as a 2D named tuple, where the fields are
+    // "l" (low) and "h" (high), each containing a 32-bit *signed* integer.
+    // In this way, the max value (0xffffffffffffffff) is represented as {"h" : -1, "l" : -1}
+    public class LongInt
+    {
```

Comment:
Our preferred style puts most opening braces on the same line as the class/if/function/etc.